src/V3Options.cpp

```diff
@@ -1812,6 +1816,10 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc,
         addIncDirUser(parseFileArg(optdir, string{valp}));
     });
 
+    if (m_systemC && m_traceFormat == TraceFormat::SAIF) {
+        fl->v3warn(E_UNSUPPORTED, "Unsupported: SAIF trace with SystemC.");
+    }
```

Comment:
What prevents this?  Most of the trace classes should work ok with SystemC.  I understand the SystemC signals won't get traced but that's true of other formats too.